README.md

```diff
@@ -148,6 +148,10 @@ The custom configuration options for the REST API are listed below:
   This defaults to `20`.
 * `IIB_PRIVILEGED_USERNAMES` - the list of users that can perform privileged actions such
   as overwriting the input index image with the built index image. This defaults to `[]`.
+* `IIB_REQUEST_LOGS_DIR` - the directory to load the request specific log files. If `None`, per
+  request log files information will not appear in the API response. This defaults to `None`.
+* `IIB_REQUEST_LOGS_DAYS_TO_LIVE` - the amount of days after which per request logs are considered
+  to be expired and may be removed. This defaults to 3.
```

Comment:
Should we default this to 5? Defaulting it to 3 would mean that logs for requests made on Friday would not be available on Monday.

Reply:
Wouldn't that increase load on the DB?

Reply:
The logs aren't stored in the database, they are just text files on the file system. I think this is fine.

Ideally, the logs would be attached to the JIRA ticket, but they may not always be. Also, the ticket may be groomed by someone that doesn't know to do that.

Reply:
I'll leave it as 3 by default and we can adjust our config as needed.